subsys/bluetooth/controller/Kconfig

```diff
@@ -470,20 +470,17 @@ config BT_CTLR_CHAN_SEL_2
 	  the Controller.
 
 config BT_CTLR_ADV_EXT
-	bool "LE Advertising Extensions" if !BT_LL_SW_SPLIT
+	bool "LE Advertising Extensions"
 	depends on BT_CTLR_ADV_EXT_SUPPORT
 	select BT_CTLR_CHAN_SEL_2 if BT_LL_SW_SPLIT && BT_BROADCASTER
 	select BT_CTLR_SCAN_REQ_NOTIFY if BT_LL_SW_SPLIT && BT_BROADCASTER
+	select EXPERIMENTAL if BT_LL_SW_SPLIT
```

Comment:
I think this changes more than you expected. 

Didn't the first instance of `BT_CTLR_ADV_EXT` simply missing `[EXPERIMENTAL]` and `select EXPERIMENTAL`?

The option should be considered experimental, regardless of `BT_LL_SW_SPLIT` AFAIK. 

This comment applies to all the changes in this PR.